packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -332,11 +332,14 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
       }
     }
 
+    // TODO: Investigate why this fails for some packages: wco.supportES2015 ? 6 : 5;
+    const terserEcma = 5;
+
     const terserOptions = {
       warnings: !!buildOptions.verbose,
       safari10: true,
       output: {
```

Comment:
Shouldn't this be `  const terserOptions = { ecma:	
        wco.supportES2015 &&	
        (!differentialLoadingNeeded || (differentialLoadingNeeded && fullDifferential))	
          ? 6	
          : 5}` to actually restory the previous behaviour?

Reply:
Somethings going wrong when its set to 6;  it was just rarely used before outside of differential loading.  The CLI also doesn't have direct control of the terser version in this situation (its transitive via `terser-webpack-plugin`) so I think its overall safer to avoid the setting completely for now.